### ISSUE TITLE
Cannot install or upgrade GUI apps due to xsetroot error

### ISSUE THREAD

**Issue by vincowl** [skycocker/chromebrew#3044]
## Description
Several apps cannot be installed or upgraded due to xsetroot compilation error. Major failure !!

If you executed a specific command:
```shell
your command here
```

Output:
```shell
output
of
your
command
here
```

## Versions and system information
chromebrew: `crew --version`

ChromeOS: `cat /etc/lsb-release`

Model: YOUR MODEL HERE

Architecture: `uname -m`

## Addtional information
Mention things we might need to know. Like:

What I think needs to be done:
- [ ] step one
- [ ] step two

---

## That's it
Thank you for submitting your issue.
When done, please delete the parts of this template which you don't need or these, which are only for guidance.

**Comment by skiman6010** [comment]
Can you edit your issues with your Version and system information? Also what specific commands are you running?

Thank you!

**Comment by vincowl** [comment]
Yep... I am just doing it right now. I needed the issue reference before. Indeed, not the cleanest way to report errors

**Comment by vincowl** [comment]
Here is the config.log from 
``` shell
crew install --keep xsetroot
```
[config.log](https://github.com/skycocker/chromebrew/files/2719625/config.log)

**Comment by skiman6010** [comment]
Only the stable channel is supported by `crew`. 

```
Q: I'm having miscellaneous errors when installing packages on the beta, dev, or canary channels. What do I do?

A: Installing Chromebrew on the beta, dev, or canary channels is NOT supported. Revert back to the stable channel and try again.```

Please track issue #2890 for reasons why the Chrome OS 72 is broken right now.

**Comment by vincowl** [comment]
Damn !! Didn't get this info... Thanks ! Going to revert to stable channel and test this. #2890 is pretty clear.... Any idea if dev channel or beta one may be supported one day ? Don't want to spend too much time recompiling low level libraries : I will have to make choices between dev channel or chromebrew.  ...sigh...

**Comment by skiman6010** [comment]
Most of the time the dev and beta channel works, it's just when Google makes a change we don't account for it until it reaches stable. It's just easier since none of us get paid for this.
